keepassxc-browser/content/autocomplete.js

```diff
@@ -227,9 +227,18 @@ kpxcAutocomplete.updatePosition = function(inputField, elem) {
     }
 
     const rect = inputField.getBoundingClientRect();
-    div.style.top = Pixels((rect.top + document.scrollingElement.scrollTop) + inputField.offsetHeight);
-    div.style.left = Pixels((rect.left + document.scrollingElement.scrollLeft));
     div.style.minWidth = Pixels(inputField.offsetWidth);
+    const bodyrect = document.body.getBoundingClientRect();
```

Comment:
Just a tiny suggestion: use `bodyRect` and `bodyStyle` instead of all-lowercase variables. Otherwise this is good to go. Nice work!

Reply:
You are right, it's better.
Fixed.